doc/api/errors.md

```diff
@@ -441,86 +441,98 @@ checks or `abort()` calls in the C++ layer.
-* `dest` {Buffer} When reporting a file system error, the `dest` will identify
-  the file path destination (if any).
+
+#### error.address
+
+* {string}
+
+If present, `error.address` is a string describing the address to which a
+network connection failed.
 
 #### error.code
 
 * {string}
 
-The `error.code` property is a string representing the error code, which is
-typically `E` followed by a sequence of capital letters.
+The `error.code` property is a string representing the error code. `error.code`
+is the most stable way to identify an error. It will only change between major
+versions of Node.js. In contrast, `error.message` strings may change between any
+versions of Node.js.
+
+#### error.dest
+
+* {string}
+
+If present, `error.dest` is the file path destination when reporting a file
+system error.
 
 #### error.errno
 
 * {string|number}
 
-The `error.errno` property is a number or a string.
-The number is a **negative** value which corresponds to the error code defined
-in [`libuv Error handling`]. See `uv-errno.h` header file
+The `error.errno` property is a number or a string. If it is a number, it is a negative value which corresponds to the error code defined in
+[`libuv Error handling`]. See `uv-errno.h` header file
 (`deps/uv/include/uv-errno.h` in the Node.js source tree) for details. In case
 of a string, it is the same as `error.code`.
 
-#### error.syscall
+#### error.info
 
-* {string}
+* {Object}
 
-The `error.syscall` property is a string describing the [syscall][] that failed.
+If present, `error.info` is an object with details about the error condition.
 
-#### error.path
+#### error.message
 
-* {string}
+* {string} 
 
-When present (e.g. in `fs` or `child_process`), the `error.path` property is a
-string containing a relevant invalid pathname.
+`error.message` is a system-provided human-readable description of the error.
+It is subject to change between versions of Node.js. Use `code` for a more
+stable indicator of the type of error.
 
-#### error.address
+#### error.path
 
 * {string}
 
-When present (e.g. in `net` or `dgram`), the `error.address` property is a
-string describing the address to which the connection failed.
+If present, `error.path` is a string containing a relevant invalid pathname.
 
 #### error.port
 
 * {number}
 
-When present (e.g. in `net` or `dgram`), the `error.port` property is a number
-representing the connection's port that is not available.
+If present, `error.port` is the network connection port that is not available.
+
+#### error.syscall
+
+* {string}
+
+The `error.syscall` property is a string describing the [syscall][] that failed.
 
 ### Common System Errors
 
-This list is **not exhaustive**, but enumerates many of the common system
-errors encountered when writing a Node.js program. An exhaustive list may be
-found [here][online].
+This is a list of system errors commonly-encountered when writing a Node.js
+program. For a comprehensive list, see the [`errno`(3) man page][].
```

Comment:
Now come to think of it, this seems to be a bit misleading since this list only includes the native system error codes, not our home-grown error codes - so, one may simply skip to the man page without scrolling down to see the `ERR_` list..but, it was originally written that way so ¯\\_(ツ)_/¯

Reply:
Moved the stability note to the general class. Hope that helps.